src/components/SelectSearch.test.tsx

```diff
@@ -1,15 +1,39 @@
 import * as React from 'react';
-import { shallow } from 'enzyme';
+import { mount } from 'enzyme';
 import SelectSearch, { SelectSearchProps } from './SelectSearch';
+import Config from '../providers/ConfigProvider/configContext';
+import { SearchResult } from '../services/ComponentMap';
+import { GROUP_PLACEHOLDER, GROUP_NOT_FOUND } from './form/constants';
 
-const selectSearchProps: SelectSearchProps = {
-    url: 'https://some-url.com',
-    name: 'name',
-    resultType: 'resultType'
-};
+describe('SelectSearch >', () => {
```

Comment:
adding a more appropriate test for `SelectSearch` now that I'm familiar with its purpose